TestPHCNoHealthyEndpoints flakytest

```
--- FAIL: TestPHCNoHealthyEndpoints (5.10s)
    healthy_endpoints_test.go:285: total requests: 2343, count200: 0, count504: 2343, failedRequests: 2343.000000
    healthy_endpoints_test.go:290: 
        	Error Trace:	/home/runner/work/skipper/skipper/proxy/healthy_endpoints_test.go:290
        	            				/home/runner/work/skipper/skipper/metrics/metricstest/metricsmock.go:33
        	            				/home/runner/work/skipper/skipper/proxy/healthy_endpoints_test.go:289
        	Error:      	Max difference between 9372 and 5925 allowed is 2811.6, but difference was 3447
        	Test:       	TestPHCNoHealthyEndpoints
```

https://github.com/zalando/skipper/actions/runs/10349016650/job/28642519098

--- a/proxy/healthy_endpoints_test.go
+++ b/proxy/healthy_endpoints_test.go
@@ -275,7 +275,7 @@ func TestPHCNoHealthyEndpoints(t *testing.T) {
 	)
 
 	servicesString := setupServices(t, healthy, unhealthy)
-	mockMetrics, ps := setupProxy(t, fmt.Sprintf(`* -> backendTimeout("20ms") -> <random, %s>`,
+	mockMetrics, ps := setupProxy(t, fmt.Sprintf(`* -> backendTimeout("20ms") -> <roundRobin, %s>`,
 		servicesString))
 	va := fireVegeta(t, ps, 5000, 1*time.Second, 5*time.Second)
 

